### PR TITLE
Fixes #12514 - add centos 7 to the version string

### PR DESCRIPTION
**Pull request by brandonweeks**

### CODE REVIEW

**Comment by brandonweeks** [comment]
Somehow I managed to goof up the most basic PR ever. Updating...

File: app/models/compute_resources/foreman/model/vmware.rb [hunk 1]
```diff
@@ -168,7 +168,7 @@ def guest_types_descriptions
         "rhel7Guest" => "Red Hat Enterprise Linux 7 (32-bit)",
         "rhel7_64Guest" => "Red Hat Enterprise Linux 7 (64-bit)",
         "centosGuest" => "CentOS 4/5/6 (32-bit)",
-        "centos64Guest" => "CentOS 4/5/6 (64-bit)",
+        "centos64Guest" => "CentOS 4/5/6/7 (64-bit)",
         "oracleLinux64Guest" => "Oracle Linux 4/5/6 (64-bit)",
```
Comment (domcleal) [inline app/models/compute_resources/foreman/model/vmware.rb]: Probably also the same here?

**Comment by domcleal** [comment]
Just wondering, does vSphere have this updated too?

**Comment by mmoll** [comment]
I think somebody[tm] should just update that mapping list for vSphere 6.0 once again... I can do that over the weekend, if nobody else steps up.

**Comment by brandonweeks** [comment]
@domcleal yes, it's updated as of VMware 6. 

@mmoll I'll try to make time today to get the rest of them.

**Comment by dLobatog** [comment]
@brandonweeks ping?

**Comment by brandonweeks** [comment]
@dLobatog pong. Diff updated with all the changes I could spot. I'm using the descriptions seen in the web UI from a vSphere 6.0U1 server, the descriptions from the API docs are hilariously wrong.

> Windows 8 Server (64 bit)

:disappointed:

**Comment by dLobatog** [comment]
@brandonweeks Thanks that's fine :smile: - I'll merge if Jenkins tests pass

**Comment by dLobatog** [comment]
@brandonweeks Hm.. this failed on Rubocop/MethodLength, the method is too long. I'll disable it inline on merge, it's one of the few reasonable cases where I think disabling the cop for that method would make sense.

**Comment by dLobatog** [comment]
Merged as d8d947d410701aefa673f1e16948e8abde32836e, thanks @brandonweeks !

**Pull request closed without merging**